src/ui/components/DialogManager/DialogManagerContainer.tsx

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 
+import {useHotkeysContext} from 'react-hotkeys-hook';
 import {useDispatch, useSelector} from 'react-redux';
-import type {DatalensGlobalState} from 'ui';
-import {useEffectOnce} from 'ui';
-import {closeDialog} from 'ui/store/actions/dialog';
-import history from 'ui/utils/history';
+
+import type {DatalensGlobalState} from '../../';
+import {useEffectOnce} from '../../';
+import {useSetHotkeysScope} from '../../hooks/useSetHotkeysScope';
+import {closeDialog} from '../../store/actions/dialog';
+import history from '../../utils/history';
 
 import DialogManager from './DialogManager';
 
 export const DialogManagerContainer = () => {
     const dispatch = useDispatch();
     const dialogs = useSelector((state: DatalensGlobalState) => state.dialog.dialogs);
 
+    const hotkeysContext = useHotkeysContext();
```

Comment:
Do we plan to change hotkeyContext? Why not move `const hotkeysContext = useHotkeysContext();` inside useSetHotkeysScope ?